src/test/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/ProfileTest.java

```diff
@@ -0,0 +1,61 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.google.common.collect.BoundType;
+import com.google.common.collect.Range;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.LinearAVF;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ProfileTest {
+
+  Profile profile;
+
+  /** Inits the profile object before each tests */
+  @BeforeEach
+  void initEach() {
+    Profile.Builder profileBuilder = new Profile.Builder();
+    LinearAVF lavf =
+        new LinearAVF.Builder()
+            .setTeleValueFunction(true)
+            .setTerraceValueFunction(true)
+            .setWifiValueFunction(true)
+            .setFloorAreaTerraceValueFunction(10d, 20d)
+            .setFloorAreaValueFunction(50d, 100d)
+            .setNbBathroomsValueFunction(2, 4)
+            .setNbBedroomsValueFunction(2, 4)
+            .setNbSleepingValueFunction(2, 4)
+            .setNbMinNightValueFunction(5, 10)
+            .setPricePerNightValueFunction(30d, 60d)
+            .build();
+    profileBuilder.setLinearAVF(lavf);
+    for (Criterion c : Criterion.values()) {
+      profileBuilder.setWeightRange(
+          c,
+          Range.range(Double.valueOf(0d), BoundType.CLOSED, Double.valueOf(10d), BoundType.CLOSED));
```

Comment:
Il me semble possible d’utiliser un constructeur plus concis. OPT.

Reply:
Fixed in commit 83b8dbb